igvc_navigation/launch/octomap.yaml

```diff
@@ -0,0 +1,76 @@
+octree:
+    resolution: 0.2 # width & length of each grid cell in m
+    clamping:
+        max: 0.95 # Value of octree considered 100% occupied stable
+        min: 0.05 # Value of octree considered 0% occupied stable
+# P(x | x) where x is either miss or hit. So prob_miss: 0.2 means that I'm 20% confident a miss is a miss
```

Comment:
uh what? does x != x. define your variables because right now they are the same.

Reply:
True. this comment is very misleading.

Reply:
You can probably just write it out.

so P(s is occupied | measurement saying s is occupied), s is an occupancy grid location

Reply:
yeah, I changed it to p(m | z).